src/components/ErrorSMS/ErrorSMS.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+
+type Props = {
+  message: string;
+  onClose: () => void;
+};
+
+export const ErrorSMS: React.FC<Props> = ({ message, onClose }) => {
```

Comment:
It's not SMS, use the better name
```suggestion
export const ErrorNotification: React.FC<Props> = ({ message, onClose }) => {
```